internal/config/parser/known.go

```diff
@@ -23,6 +23,7 @@ var known = [][]string{
 	{"AWS.REGION", ""},
 	{"AWS.SECRET", ""},
 	{"AWS.SESSION", ""},
+	{"GCP.APPCRDS", "GOOGLE_APPLICATION_CREDENTIALS"},
```

Comment:
@ezr-ondrej Not sure about that

Reply:
depends if we want to rename this variable.

Reply:
`GOOGLE_CREDENTIALS` sounds good enough to me tho :) or even `GCP_CREDENTIALS` and we could have `gcp.credentials` and
```
gcp:
  credentials
```

And we need no renaming :)

Reply:
```golang	
{"GCP.CREDENTIALS", "GOOGLE_CREDENTIALS"},
```